frontend/src/Layout/Footer.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import classes from "./Footer.module.css";
+import HeaderClasses from "./Header.module.css";
+import { BsGithub } from "react-icons/bs";
+import Grid from "@mui/material/Unstable_Grid2";
+import Item from "@mui/material/Unstable_Grid2";
+
+const Footer = () => {
+  return (
+    <footer className={classes.footer}>
+      {/* Contact */}
+      <section className={classes.contact_buttons}>
+        <div>
+          <span>Visit our project respository: </span>
+        </div>
+
+        <div>
+          <a
+            href="https://github.com/joans/teachme"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <span>
+              {" "}
+              <BsGithub />
+            </span>
+          </a>
+        </div>
+      </section>
+      {/* End contact */}
+
+      {/* Footer navigation */}
+      <Grid container spacing={2} className={classes.footer_nav}>
```

Comment:
Das Grid funktioniert bei mir nicht. Ich hab dir ja schon im Chat geschrieben diesbezüglich. Du brauchst eigentlich auch gar kein Grid in dieser Komponente.